src/win32/fs.cpp

```diff
@@ -1455,7 +1455,7 @@ WinDirNotify::~WinDirNotify()
 #endif
 }
 
-FileAccess* WinFileSystemAccess::newfileaccess(bool followSymLinks = true)
+FileAccess* WinFileSystemAccess::newfileaccess(bool followSymLinks)
```

Comment:
why? it doesn't matter since it's unused, but I can't help to wonder why the change? ;-)

Reply:
The MSVC compiler won't accept that on the function definition, only on the declaration, even if it matches the declaration exactly.  It throws an error, otherwise I would have left it :-)  cheers